Source/Csla.Shared/Server/SimpleDataPortal.cs

```diff
@@ -203,8 +203,8 @@ private async Task<DataPortalResult> Execute(DataPortalTarget obj, DataPortalCon
         }
         throw DataPortal.NewDataPortalException(
               "DataPortal.Execute " + Resources.FailedOnServer,
-              new DataPortalExceptionHandler().InspectException(obj.GetType(), obj, null, "DataPortal.Execute", ex),
-              obj);
+              new DataPortalExceptionHandler().InspectException(obj.Instance.GetType(), obj.Instance, null, "DataPortal.Execute", ex),
```

Comment:
@SachinPNikam do we need to confirm that `obj.Instance` isn't null?

Reply:
I tested locally it's not null ,it is actual instance of Command base BO which the exception.